Common/Brokerages/BitfinexBrokerageModel.cs

```diff
@@ -0,0 +1,50 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using QuantConnect.Orders;
+using QuantConnect.Securities;
+using QuantConnect.Orders.Fills;
+using QuantConnect.Orders.Fees;
+using System.Linq;
+
+namespace QuantConnect.Brokerages
+{
+    /// <summary>
+    /// Provides Bitfinex specific properties
+    /// </summary>
+    public class BitfinexBrokerageModel : DefaultBrokerageModel
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BitfinexBrokerageModel"/> class
+        /// </summary>
+        /// <param name="accountType">The type of account to be modelled, defaults to <see cref="AccountType.Margin"/></param>
+        public BitfinexBrokerageModel(AccountType accountType = AccountType.Margin)
+            : base(accountType)
+        {
+        }
+
+        /// <summary>
+        /// Provides Bitfinex fee model
+        /// </summary>
+        /// <param name="security"></param>
+        /// <returns></returns>
+        public override IFeeModel GetFeeModel(Security security)
+        {
+            return new BitfinexFeeModel();
+        }
+    }
```

Comment:
We might also want to override `DefaultMarkets` here, since the default market for `SecurityType.Crypto` is `Market.GDAX` and this would require algorithms to pass `Market.Bitfinex` to all `AddCrypto` calls in `Initialize`.

Maybe something similar to this:
``` C#
        public override IReadOnlyDictionary<SecurityType, string> DefaultMarkets {  get; } = GetDefaultMarkets();

        private static IReadOnlyDictionary<SecurityType, string>  GetDefaultMarkets()
        {
            var map = DefaultMarketMap.ToDictionary();
            map[SecurityType.Crypto] = Market.Bitfinex;
            return map.ToReadOnlyDictionary();
        }
```

Reply:
I remember Jared mentioned about this, but I thought that it's managed internally, but not on current brokerage implementation. Will do.